README.md

```diff
@@ -1,7 +1,238 @@
+it appeared in the desired way on different devices.
+- I also wrote automated test for validators.py file. Test is located in tests.py file.
+- I continually  tested code in the [W3C HTML Validator](https://validator.w3.org/)
+    in order to get rid of errors. 
+- Vocabulary and grammar was checked on [Spell checker](http://www.reverso.net/spell-checker/english-spelling-grammar/)
+- [Am I Responsive](http://ami.responsivedesign.is/#) - Online tool was used to display the project on various devices;
+Image available  in wireframes.
+
+
+
+ERRORS:
+/favicon.ico:1 Failed to load resource: the server responded with a status of 404 ()- the only one that comes in console.
+
+## **7.1 Code Validation
+
+- I used the [W3C HTML Validator tool](https://validator.w3.org/#validate_by_input) to validate my HTML code.
+    - The W3C Validator tool doesn't recognise the Jinja templating, which has resulted in it showing a lot of errors in relation to the Jinja code. However, all other code is validating fine.
+- I used the [W3C CSS Validator tool](https://jigsaw.w3.org/css-validator/#validate_by_input) to validate my CSS code.
+- I used the [Pep8 Online tool](http://pep8online.com/) to validate my Python syntax.
+- I used CLI to sort all my imports by using command  "isort -rc ."
+- I used CLI to make sure line length is not going above 79 in one line by using command  "black --line-length=79"
+- I used CLI to check all the errors in the code by using command  "flake8"
+
+## ** 7.2 Pagination Test (manual)
+
+1. Test the pagination page by page and verify that the same amoutn of recipes are always appearing on the same page number
+2. Test the pagination next / previous page functionality and verify that the same recipes are always appearing on the same page number
+3. Test the 'next' button to check if disappear  on last page, and 'prevoius' button if disappear  on first page.
+
+
+# 8. DEPLOYMENT
+
+## 8.1 DEPLOYING FROM GITHUB 
+
+1. Log onto Github
+2. Select the respository you want to deploy
+3. On the repository page, click on "Settings" and scroll down to "Github Pages"
+4. From the "Source" dropdown select "Master Branch" and click "Save"
+5. The message "Your site is ready to be published at https://username.github.io/Repository Name/" will 
+   appear under Github Pages
+
+## 8.2 CLONING FROM GITHUB 
+
+1. Follow this link to my [Project Repository on Github](https://github.com/tomciosegal/fitness_pot)
+2. On the repository page click "Clone or Download"
+3. In the Clone with HTTPs section, copy the clone URL for the repository.
+4. In your local IDE open Git Bash.
+5. Change the current working directory to the location where you want the cloned directory to be made.
+6. Type git clone, and then paste the URL you copied in Step 3 - 
+    "git clone https://github.com/tomciosegal/fitness_pot/"
+7. Press enter and your local clone will be created.
+ 
+
+## 8.3 DEPLOYING TO HEROKU 
+
+1. Type 'heroku ps:scale web=1' into bash terminal
```

Comment:
State that this project was deployed to Heroku and include the link to the live site